Gruntfile.js

```diff
@@ -0,0 +1,180 @@
+module.exports = function(grunt) {
```

Comment:
Could `saucelabs` be launched without `grunt`? Some config for example and npm scripts?